### ISSUE TITLE
Error: Module Not Found

### ISSUE THREAD

**Issue by dhaganjet** [crashmax-dev/fireworks-js#110]
### Description

When I do ng serve in my project, it generates an error. My angular is version 13.3.0

Error message below:

```
./src/app/app.module.ts:42:0-82 - Error: Module not found: Error: Package path ./lib/ng-fireworks.module is not exported from package C:\Dev-world\new-website\node_modules\@fireworks-js\angular (see exports field in C:\Dev-world\new-website\node_modules\@fireworks-js\angular\package.json)

Error: node_modules/@fireworks-js/angular/lib/ng-fireworks.directive.d.ts:11:18 - error TS2314: Generic type 'ɵɵDirectiveDeclaration' requires 6 type argument(s).

11     static ɵdir: i0.ɵɵDirectiveDeclaration<FireworksDirective, "[ngFireworks], ng-fireworks", ["ngF

11     static ɵdir: i0.ɵɵDirectiveDeclaration<FireworksDirective, "[ngFireworks], ng-fireworks", ["ngFireworks"], { "options": "options"; }, {}, never, never, false>;
```

Does anyone know what the issue is?

### Which package are you using?

@fireworks-js/angular

### fireworks-js version

2.10.5

### Browser

Chrome Version 114.0.5735.134 (Official Build) (64-bit)

**Comment by crashmax-dev** [comment]
@dhaganjet Hi, how can i reproduce?

**Comment by crashmax-dev** [comment]
Look at how the example works https://github.com/crashmax-dev/fireworks-js/tree/master/examples/with-angular/src/app

**Comment by dhaganjet** [comment]
Does it work for angular 13?

**Comment by dhaganjet** [comment]
> @dhaganjet Hi, how can i reproduce?

Here are the steps I took
1. Installed angular with "npm install @fireworks-js/angular --legacy-peer-deps"
2. added "import { NgFireworksModule } from '@fireworks-js/angular/public-api'" to the app.component.module
3. added "NgFireworksModule" to the imports array

And when i run "ng serve" it generates the error

**Comment by crashmax-dev** [comment]
> 

Right now angular ^14 is specified in peerDependencies. Maybe we can change the build and lower the version below 14 🤔

**Comment by dhaganjet** [comment]
> Right now angular ^14 is specified in peerDependencies. Maybe we can change the build and lower the version below angular 14 🤔

Okay let's try that and see if it works

**Comment by crashmax-dev** [comment]
@dhaganjet Hi, you need use import https://github.com/crashmax-dev/fireworks-js/blob/811205e2b9fb8df5edc2678bce9a2091be429ad3/examples/with-angular/src/app/app.module.ts#L3 not from `@fireworks-js/angular/public-api`

**Comment by crashmax-dev** [comment]
> Right now angular ^14 is specified in peerDependencies. Maybe we can change the build and lower the version below angular 14 thinking

@dhaganjet fixed in #111

**Comment by dhaganjet** [comment]
I decided updated my project to angular 16. It works now. Thanks
